slack-api-client/src/main/java/com/slack/api/socket_mode/impl/SocketModeClientTyrusImpl.java

```diff
@@ -213,8 +213,8 @@ public boolean verifyConnection() {
                         this.currentSession.getId(),
                         e.getMessage());
             }
-            if (getLogger().isDebugEnabled()) {
-                getLogger().debug("Failed to receive a pong message: {}", ping);
+            if (getLogger().isTraceEnabled()) {
+                getLogger().trace("Failed to receive a pong message: {}", ping);
             }
```

Comment:
Consider making this a warning.

```suggestion
            getLogger().warn("Failed to receive a pong message: {}", ping);
```